bodhi/server/templates/master.html

```diff
@@ -24,8 +24,10 @@
     % else:
         <title>${update.title} - Fedora Updates System</title>
     % endif
-    <link href="${request.static_url('bodhi:server/static/bootstrap/css/bootstrap.css') }" rel="stylesheet" />
-    <link href="${request.static_url('bodhi:server/static/font-awesome-4.4.0/css/font-awesome.min.css') }" rel="stylesheet" />
+    <link href="https://apps.fedoraproject.org/global/fedora-bootstrap-1.0.1/fedora-bootstrap.css" rel="stylesheet" />
+    <link rel='stylesheet' href='//cdn.jsdelivr.net/font-hack/2.020/css/hack.min.css'>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,700,700italic' rel='stylesheet' type='text/css'>
+    <link href="https://ryanlerch.fedorapeople.org/font-awesome.css" rel="stylesheet" />
```

Comment:
Before we merge this, I think we should try to get these css files packaged for Fedora, or at least document what they are (version/license/upstream) and put them in Bodhi's server/static folder so we don't depend on external services. Some of these things get blocked by Privacy Badger (like the googleapis.com link).

I don't mind taking on the packaging work if you don't want to do that.